monero/docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: monero_server_1
+      APP_PORT: 9976
+
+  server:
+    image: deverickapollo/umbrel-monero:v1.0.2@sha256:ceb4f8137cb6c1db391e6df40d3c7790e1e3cc251d344030cc44f2ec378f2521
+    depends_on: [monerod]
+    restart: on-failure
+    expose:
+      - "9976"
+    volumes:
+      - ${APP_DATA_DIR}/data/app:/data # volume to persist advanced settings json
+      - ${APP_MONERO_DATA_DIR}:/monero/.monero # volume to persist bitmonero.conf
+    environment:
+      PORT: "9976"
+      MONERO_HOST: "${APP_MONERO_NODE_IP}"
+      MONERO_P2P_PORT: "${APP_MONERO_P2P_PORT}"
+      MONERO_RPC_PORT: "${APP_MONERO_RPC_PORT}"
+      MONERO_RESTRICTED_RPC_PORT: "${APP_MONERO_RESTRICTED_RPC_PORT}"
+      MONERO_DEFAULT_NETWORK: "${MONERO_DEFAULT_NETWORK}"
+      MONERO_RPC_USER: "${APP_MONERO_RPC_USER}"
+      MONERO_RPC_PASSWORD: "${APP_MONERO_RPC_PASS}"
+      MONERO_RPC_HIDDEN_SERVICE: "${APP_MONERO_RPC_HIDDEN_SERVICE}"
+      MONERO_P2P_HIDDEN_SERVICE: "${APP_MONERO_P2P_HIDDEN_SERVICE}"
+      DEVICE_DOMAIN_NAME: "${DEVICE_DOMAIN_NAME}"
+      MONEROD_IP: "${APP_MONERO_NODE_IP}"
+      TOR_PROXY_IP: "${APP_MONERO_TOR_PROXY_IP}"
+      TOR_PROXY_PORT: "9050"
+      TOR_PROXY_CONTROL_PORT: "9051"
+      TOR_PROXY_CONTROL_PASSWORD: "moneyprintergobrrr"
+      I2P_DAEMON_IP: "${APP_MONERO_I2P_DAEMON_IP}"
+      I2P_DAEMON_PORT: "7656"
+    networks:
+      default:
+        ipv4_address: $APP_MONERO_IP
+
+  monerod:
+    user: "1000:1000"
+    restart: unless-stopped
+    stop_grace_period: 15m
+    command: "${APP_MONERO_COMMAND}"
+    image: sethsimmons/simple-monerod:v0.18.3.1@sha256:1ef52fe371a8085c1cf8ba579610f0a33537a170561f2f1370d4f528055f170f
```

Comment:
This just needs to be changed to the multi-arch digest so it works on arm64 and amd64. I get this:

```
$ sudo docker pull sethsimmons/simple-monerod:v0.18.3.1
v0.18.3.1: Pulling from sethsimmons/simple-monerod
Digest: sha256:294f8460d63240ba199e63c52cbd01ab71fbd286c2af62a19ae8a73a302afabd
```